src/lib/peer_trust/peer_trust.mli

```diff
@@ -68,17 +72,18 @@ end
 (* FIXME The parameter docs don't render :( *)
 
 (** Instantiate the module.
-    @param Peer The identifiers for peers
+    @param Peer The identifiers for peers. N.b. this should be something that is
```

Comment:
Can you call this `Peer_identifier` (or some other name), it may be too easy for a reader or someone who is modifying this later to miss this comment and assume that this is (or should be) the same as the existing Peer record in `network_peer`.